python/graphscope/dataset/ldbc.py

```diff
@@ -270,17 +271,53 @@ def load_ldbc(sess, prefix, directed=True):
 """
 
 
-def load_ldbc(sess, prefix, directed=True):
+def load_ldbc(sess, prefix=None, directed=True):
     """Load ldbc dataset as a ArrowProperty Graph.
+
     Args:
         sess (:class:`graphscope.Session`): Load graph within the session.
-        prefix (str): Data directory.
+        prefix: `PathLike` object that represents a path.
+            With standalone mode, set prefix None will try to download from
+            source URL. Defaults to None.
         directed (bool, optional): Determine to load a directed or undirected graph.
             Defaults to True.
     Returns:
-        :class:`graphscope.Graph`: A Graph object which graph type is ArrowProperty
+        :class:`graphscope.framework.graph.GraphDAGNode`:
+            A Graph node which graph type is ArrowProperty, evaluated in eager mode.
+
+    Examples:
+        .. code:: python
+
+        >>> # lazy mode
+        >>> import graphscope
+        >>> from graphscope.dataset.ldbc import load_ldbc
+        >>> sess = graphscope.session(mode="lazy")
+        >>> g = load_ldbc(sess, "/path/to/dataset", True)
+        >>> g1 = sess.run(g)
+
+        >>> # eager mode
+        >>> import graphscope
+        >>> from graphscope.dataset.ldbc import load_ldbc
+        >>> sess = graphscope.session(mode="eager")
+        >>> g = load_ldbc(sess, "/path/to/dataset", True)
+
     """
-    prefix = os.path.expandvars(prefix)
+    if prefix is not None:
+        prefix = os.path.expandvars(prefix)
+    else:
+        fname = "ldbc_sample.tar.gz"
+        origin = (
+            "https://graphscope.oss-cn-beijing.aliyuncs.com/dataset/ldbc_sample.tar.gz"
```

Comment:
We could add a custom domain name for the dataset, e.g., `https://data.graphscope.io`, you may find the DNS settings on aliyun webpage to see how to set custom domain name for oss buckets.